cloud/shared/bin/lib/variable_definition_loader.py

```diff
@@ -2,26 +2,21 @@
 import json
 
 
-# Loads all configuration variable definition files
-#
-# Requires that:
-#   - Each variable definition file is referenced in
-#     def load_repo_variable_definitions_files():
 class VariableDefinitionLoader:
+    """ 
+    Loads all variables from a valid variable definition json file.
+    """
 
     def __init__(self, variable_definitions={}):
         self.variable_definitions: dict = variable_definitions
 
     def load_definition_file(self, definition_file_path: str):
         with open(definition_file_path, "r") as file:
+            # json.loads() returns a dictionary and thus removes duplicates. 
```

Comment:
Does it keep the last one it sees or the first one?

Reply:
It keeps the last one (comment updated)